nilearn/decoding/searchlight.py

```diff
@@ -176,7 +177,7 @@ def _group_iter_search_light(list_rows, estimator, X, y,
                 else:
                     crlf = "\n"
                 percent = float(i) / len(list_rows)
-                percent = round(percent * 100, 2)
+                percent = np.round(percent * 100, 2)
```

Comment:
My understanding is the same.  I replaced it here while replacing in other places, as it seemed more robust to future changes and a good habit to simply use `np.round` in any code already importing numpy.

Happy to change it back.